cmd/kubernetes-static/main.go

```diff
@@ -179,18 +180,18 @@ func main() {
 		},
 	}
 
-	ksmClient, err := ksm.NewKSMClient(log.NewStdErr(true))
+	ksmClient, err := client.NewKSMClient(log.NewStdErr(true))
 	if err != nil {
 		log.Fatal(err)
 	}
-	ksmGrouperConfig := &ksm.GrouperConfig{
+	ksmGrouperConfig := &grouper.Config{
```

Comment:
here the same for the grouper, when we are going to have multiple things in the same main just "client" and "grouper" is going to be confusing

Reply:
Definitely. My idea is that `grouper` should be purely internal and not exposed outside of the `ksm` package, but of course is still there in the old main...
I'll rename the import but hopefully won't be necessary in the future.

Reply:
Import renamed